wakuv2/waku.go

```diff
@@ -1347,7 +1351,7 @@ func (w *Waku) OnNewEnvelopes(envelope *protocol.Envelope, msgType common.Messag
 		trouble = true
 	}
 
-	common.EnvelopesValidatedCounter.Inc()
+	common.EnvelopesValidatedCounter.With(prometheus.Labels{"pubsubTopic": envelope.PubsubTopic(), "type": msgType}).Inc()
```

Comment:
Note that we need the meaning of `common.MissingMessageType` to be consistent moving forward, otherwise it can mess up our metrics. i.e. only messages returned by the periodic store query can have this type